### ISSUE TITLE
fatal error: 'clang/Driver/Options.inc' file not found

### ISSUE THREAD

**Issue by jroelofs** [llvm/llvm-project#64882]
When building `lib/StaticAnalyzer/Frontend/FrontendActions.cpp` in Stage2 of a bootstrapped modules build, we see:

```
llvm-project/clang/include/clang/Driver/Options.h:49:10: fatal error: 'clang/Driver/Options.inc' file not found
   49 | #include "clang/Driver/Options.inc"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by jroelofs** [comment]
Fixed by 1e4d6122cda6529781ecf467c2ae84e5dd41acdf

**Comment by EugeneZelenko** [comment]
/cherry-pick https://github.com/llvm/llvm-project/commit/1e4d6122cda6529781ecf467c2ae84e5dd41acdf
